.jshintrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
don't think we need that tool
it's better to use `eslint`

Reply:
Sorry, I'll switch it to eslint.